SU2_CFD/include/solvers/CScalarSolver.inl

```diff
@@ -404,6 +404,73 @@ void CScalarSolver<VariableType>::ImplicitEuler_Iteration(CGeometry* geometry, C
   CompleteImplicitIteration(geometry, solver_container, config);
 }
 
+template <class VariableType>
+void CScalarSolver<VariableType>::PrepareExplicitIteration(CGeometry* geometry, CSolver** solver_container,
```

Comment:
Merge this one with ExplicitEuler_Iteration please, no need for two loops, and that makes it more similar to the flow solver version.